src/main/java/reposense/authorship/FileInfoAnalyzer.java

```diff
@@ -161,17 +158,15 @@ private void aggregateBlameAuthorModifiedAndDateInfo(RepoConfiguration config, F
         LocalDateTime sinceDate = config.getSinceDate();
         LocalDateTime untilDate = config.getUntilDate();
 
-        for (int lineCount = 0; lineCount < blameResultLines.length; lineCount += 5) {
-            String commitHash = blameResultLines[lineCount].substring(0, FULL_COMMIT_HASH_LENGTH);
-            String authorName = blameResultLines[lineCount + 1].substring(AUTHOR_NAME_OFFSET);
-            String authorEmail = blameResultLines[lineCount + 2]
-                    .substring(AUTHOR_EMAIL_OFFSET).replaceAll("<|>", "");
-            long commitDateInMs = Long.parseLong(blameResultLines[lineCount + 3].substring(AUTHOR_TIME_OFFSET)) * 1000;
-            LocalDateTime commitDate = LocalDateTime.ofInstant(Instant.ofEpochMilli(commitDateInMs),
-                    config.getZoneId());
-            Author author = config.getAuthor(authorName, authorEmail);
-
-            int lineNumber = lineCount / 5;
+        for (int lineCount = 0; lineCount < blameResultLines.length; lineCount += BLAME_LINE_INFO_ROW_COUNT) {
```

Comment:
I was originally intending for most of this logic to be encapsulated in GitBlame, largely because we don't want to run GitBlame for every line. So, ideally something like `blameFile` in `GitBlame` would output a list of `GitBlameLineInfo`. We might have to augment `GitBlameLineInfo` with additional information like line numbers.

Reply:
Hi, I have followed what you said and created a new method, blameFile, that converts the blameResult into a list of GitBlameLineInfo. The reason why I used blameLine initially, was because of its mention in issue, which got me too focused on using it. Either way, thanks for your input!

Reply:
I should have been clearer in the issue, sorry for the confusion!